BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -150,6 +138,64 @@ - (void)fireIdentifyCall {
     }
 }
 
+#pragma mark - enable push and auto identify
+- (NSString*)getLastModifiedUNAuthorizationStatus {
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    NSString* authorizationStatus = (NSString *)[defaults objectForKey:kBlueshiftUNAuthorizationStatus];
+    return  authorizationStatus;
+}
+
+- (void)setLastModifiedUNAuthorizationStatus:(NSString*) authorizationStatus {
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    [defaults setObject:authorizationStatus forKey:kBlueshiftUNAuthorizationStatus];
+}
+
+/// Check current UNAuthorizationStatus status with last Modified UNAuthorizationStatus status, if its not matching
+/// update the last Modified UNAuthorizationStatus in userdefault and fire identify call
+- (BOOL)validateChangeInUNAuthorizationStatus {
+     NSString* lastModifiedUNAuthorizationStatus = [self getLastModifiedUNAuthorizationStatus];
+    if ([[BlueShiftAppData currentAppData] currentUNAuthorizationStatus]) {
+        if(!lastModifiedUNAuthorizationStatus || [lastModifiedUNAuthorizationStatus isEqualToString:@"NO"]) {
```

Comment:
keep the hardcoded string in Blueshift / InApp Constants

Reply:
I think you have already added that key in the constant

Reply:
Fixed